backstop-non-responsive-ce.json

```diff
@@ -257,6 +257,29 @@
       "misMatchThreshold": 0,
       "requireSameDimensions": true
     },
+    {
+      "label": "carousel",
+      "cookiePath": "",
+      "url": "http://localhost:8000/tests/custom-elements/carousel.html",
+      "referenceUrl": "",
+      "readyEvent": "",
+      "readySelector": "",
+      "delay": 0,
+      "hideSelectors": [],
+      "removeSelectors": [],
+      "hoverSelector": "",
+      "clickSelector": "",
+      "postInteractionWait": "",
+      "selectors": [
+        ".test-base",
+        ".test-single",
+        ".test-dots",
+        ".test-pagination"
```

Comment:
Seems like there are 4 selectors here and in `non-responsive.json` but I'm seeing 8 new `.png`s. Are all 8 necessary?

Reply:
Will be removing duplicate ones.